test/fixtures/insecure-registries/push-to-registry-job.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: push-to-registry
```

Comment:
WDYT of `copy-dockerhub-image-to-local-registry`? I know it's long but something like this can capture the essence of what this Job is doing without having to look at the details below